apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1073,6 +1073,13 @@ type GlobalConfig struct {
 
 	// FIPS contains configuration used to customize the FIPS proxy sidecar.
 	FIPS *FIPSConfig `json:"fips,omitempty"`
+
+	// ChecksCardinality configures tag cardinality for the metrics collected by integrations (`low`, `orchestrator` or `high`).
+	// See also: https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#tags-cardinality.
+	// Not set by default to avoid overriding existing DD_CHECKS_TAG_CARDINALITY configurations, the default value in the Agent is low.
+	// Ref: https://github.com/DataDog/datadog-agent/blob/856cf4a66142ce91fd4f8a278149436eb971184a/pkg/config/setup/config.go#L625
+	// +optional
+	ChecksCardinality *string `json:"checksCardinality,omitempty"`
```

Comment:
could we call it `checksTagCardinality` instead?

i saw that it's called `checksCardinality` in the helm chart, but it might be easier to intuit its meaning and relationship to the [dogstatsd parameter](https://github.com/DataDog/datadog-operator/commit/3efc80aca617e12442ba4f6972fbfa5fdf7791a1) if `tag` is included in the name

Reply:
Adressed by [`760049b`](https://github.com/DataDog/datadog-operator/pull/1309/commits/760049b961b2c31b98be7ed7a57ab8277d410571)